### ISSUE TITLE
[$250] IOS - Login - Validation link is not received when a new user logs in

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#54463]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.77-6
**Reproducible in staging?:** Y
**Reproducible in production?:** Y 
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** htad26+ttt@gmail.com
**Issue reported by:** Applause - Internal Team

## Action Performed:
Prerequisite
Sign in to new dot on web with a new Gmail account but do not validate it
1. Navigate to hybrid iOS/android app
2. Enter the email address you used to sign in on web and click on next
3. Navigate to your Gmail inbox and wait for the validation link to arrive to your Gmail address

## Expected Result:
Validation link is received on the Gmail of the account

## Actual Result:
Validation link is not received on the Gmail of the account and user has to click on resend link to receive the validation link

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [x] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/db2d2d7f-49aa-4cbc-8000-fae754d37dfc

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021871402382458153824</li>
        <li>Upwork Job ID: 1871402382458153824</li>
        <li>Last Price Increase: 2025-01-14</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @abdulrahuman5196</details>

**Comment by RachCHopkins** [comment]
I have seen these first emails take a while. For example:

![image](https://github.com/user-attachments/assets/1ba0719e-c6aa-4ac3-9d25-e4aa087c3a8b)

This _just_ been received and it's timestamped 5 minutes ago. So I'll wait a little longer.

**Comment by RachCHopkins** [comment]
Nope, can confirm, there is no record of any email being sent with a validation code.

**Comment by bobmakesmagics** [comment]
Hi,
Is there any chance for me to work on this issue yet?

**Comment by abdulrahuman5196** [comment]
@RachCHopkins How is this external? Isn't mailing done via backend?

**Comment by abdulrahuman5196** [comment]
> @RachCHopkins How is this external? Isn't mailing done via backend?

Gentle ping

**Comment by RachCHopkins** [comment]
@abdulrahuman5196 I don't decide whether things are external or internal. Are you saying that this should be a backend issue?

**Comment by abdulrahuman5196** [comment]
> [@abdulrahuman5196](https://github.com/abdulrahuman5196) I don't decide whether things are external or internal. Are you saying that this should be a backend issue?

yes. This should be backend issue. @RachCHopkins

**Comment by RachCHopkins** [comment]
Low priority.
